src/modules/search/components/Track/Options.jsx

```diff
@@ -28,6 +33,8 @@ export default class Options extends React.Component {
 	}
 }
 
+export default onClickOutside(Options);
```

Comment:
Here's another case where we should wrap the component into a styled component and use it as a selector directly.

```suggestion
export default styled(onClickOutside(Options))``;
```

Reply:
these suggested change comments are amazing!